react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,33 @@
+
+body {
+  margin: 0 auto;
+  font-family: sans-serif, serif;
+}
+
+label {
+  font-weight: bolder;
+}
+
+.modal-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-top: 3rem;
+}
+
+.status-logo {
+  position: absolute;
+  right: 40%;
+  top: 8%;
+}
```

Comment:
Try just making it a sibling of the input. It's _very_ rare that absolute positioning is required. It's usually used when you want to take an element **out of document flow**. I don't think that's what you're trying to do.